test/controllers/premiumlease/PremiumsGrantLeaseControllerSpec.scala

```diff
@@ -146,7 +146,7 @@ class PremiumsGrantLeaseControllerSpec extends SpecBase with MockitoSugar {
 
         status(result) mustEqual SEE_OTHER
         redirectLocation(result).value mustEqual controllers.premiumlease.routes.YearLeaseAmountController
-          .onPageLoad(taxYear, NormalMode)
```

Comment:
@willjshark we can add 2 more tests here for `RentalsRentARoom`

Reply:
Fixed